bin/pybsn-schema

```diff
@@ -113,6 +113,6 @@ else:
     schema = bcf.schema(path)
 
 if args.raw:
-    print(json.dumps(schema, indent=4, cls=pybcf.BCFJSONEncoder))
```

Comment:
@andi-bigswitch, this was a leftover reference forgotten in #31 that was found by the quality gate.

Reply:
Good find! 👍 

Actually, the bug is even older than #31, `BCFJSONEncoder` was removed in 3e392567863b2a37f38dc0f9aadd4db81322fafd (on Dec 16, 2014) 🙃